spec/factories/scope_limitations.rb

```diff
@@ -1,11 +1,45 @@
 FactoryBot.define do
   factory :scope_limitation do
     sequence(:code) { |n| format('AA%03d', n) }
-    meaning { 'Meaning' }
-    description { 'Description' }
+    meaning { Faker::Lorem.sentence }
+    description { Faker::Lorem.paragraph }
     substantive { true }
     delegated_functions { false }
 
+    trait :substantive do
+      substantive { true }
+    end
+
+    trait :delegated_functions do
+      delegated_functions { true }
+    end
+
+    transient do
+      joined_proceeding_type { create :proceeding_type }
+    end
+
+    trait :substantive_default do
+      substantive { true }
+      after(:create) do |sl, evaluator|
```

Comment:
Use `scope_limitation` here as is done in `delegated_functions_default` rather than using the unnecessary abbreviation `sl`